src/to_bytes.rs

```diff
@@ -50,6 +50,55 @@ pub unsafe fn transmute_to_bytes_unchecked<S>(from: &S) -> &[u8] {
     slice::from_raw_parts(from as *const S as *const u8, size_of::<S>())
 }
 
+/// Transmute a single mutable instance of an arbitrary type into a mutable
+/// slice of its bytes.
+///
+/// # Safety
+/// 
+/// This function is very ill advised, since it can be exploited to break
+/// invariants of the source type. Any modification that leaves the data
+/// in an inconsistent state with respect to `S` is undefined behavior.
```

Comment:
"yields" or "results in" maybe? It, itself, isn't undefined behaviour, UB occurs afterward, right?